Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Find_and_replace_text_within_table
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Program program = new Program();  
+            //Load the Input document.
+            using (WordDocument document = new WordDocument(Path.GetFullPath(@"../../Input.docx"), FormatType.Docx))
+            {              
+                //Access table in Word document.
+                WTable table = document.Sections[0].Tables[0] as WTable;
+                program.FindTextWithinTable(table, document); 
+                //Save the document.
+                document.Save(Path.GetFullPath("../../Sample.docx"), FormatType.Docx);
+            }
+        }
+        /// <summary>
+        /// Find text within table in Word document.
+        /// </summary>
+        /// <param name="table"></param>
+        /// <param name="paragraph"></param>
+        public void FindTextWithinTable(WTable table, WordDocument document)
```

Comment:
Change method name as private void FindAndReplaceInTable()

Reply:
This is addressed